### DISCUSSION TITLE
Planning engine result are different on the same planning factors

### DISCUSSION THREAD

**Discussion by sgfisme** [frePPLe/frepple#512] [Q&A] [answered]
Hi, there,

We tried to drop foreign key on operationplans and move update demand plan to export 3 for exporting performance tuning.
The data count as 157505 is less 187 than original as 157692 before changed.

The two round testing we set the same planning factors and the same current date. Just wondering to know that 
the planning result are different due to drop foreign key and change exporting mechanisms? Or planning results are different on each round even if the planning factors are the same?

**Comment by jdetaeye** [comment, marked as answer]
The planning is fully deterministic. Two planning runs with the same input will give 100% the same output.

**Reply by sgfisme** [reply]
Thanks, @jdetaeye
